web/apps/adminApp/translations/locale-da.json

```diff
@@ -0,0 +1,96 @@
+{
+  "user": {
+    "action": {
+      "all": "Alle brugere",
+      "create": "Opret bruger",
+      "delete": "Slet bruger",
+      "edit": "Redigér bruger",
+      "remove_role": "Fjern rolle fra bruger",
+      "search_for_role": "Søg efter roller",
+      "view": "Se bruger"
+    },
+    "heading": "Brugere",
+    "firstname": "Fornavn",
+    "surname": "Efternavn",
+    "email": "E-mail",
+    "placeholder": {
+      "firstname": "Indtast fornavn",
+      "surname": "Indtast efternavn",
+      "email": "Indtast e-mail"
+    },
+    "messages": {
```

Comment:
Can you specify which of these are messages/inline messages (https://styleguide.os2display.dk/?p=viewall-modules-message) and which are text?